extensionMain.js

```diff
@@ -0,0 +1,42 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+'use strict'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const join = require('path').join
+
+/**
+ * This file serves as the extension's entryPoint.
+ * It loads the actual entryPoint from a webpack bundle or from
+ * tsc compiled source based on the AWS_TOOLKIT_IGNORE_WEBPACK_BUNDLE environment variable.
```

Comment:
nit: You mean `ASK_TOOLKIT_IGNORE_WEBPACK_BUNDLE`? :-)

Reply:
Ahh, good catch lol